ReleaseProcessScript/tools/Core/semver_functions.ps1

```diff
@@ -107,7 +107,7 @@ function Get-Possible-Versions-Hotfix ($Version, $MeantForCurrentVersion = $fals
     $NextPossibleFullVersion = "$($OriginalMajor).$($OriginalMinor).$($NextPatch)"  
 
     $NextPreReleaseNumber = [string](1 + $OriginalPreReleaseNumber)
-    $NextPossiblePreReleaseVersion = "$($OriginalMajor).$($OriginalMinor).$($NextPatch)-$($PreReleaseType).$($NextPreReleaseNumber)" 
+    $NextPossiblePreReleaseVersion = "$($OriginalMajor).$($OriginalMinor).$($OriginalPatch)-$($PreReleaseType).$($NextPreReleaseNumber)" 
```

Comment:
shouldn't there be a test for this in the second commit?

Reply:
The added tests are in the first commit. Do you want me to shift them to the second one?

Reply:
yes, please. the first commit was just refactoring, so there shouldn't be behavioral changes and all tests green. The added test can't be green without the fix, so it belongs to the second commit. alternatively, you could add the test in the test in the first commit with the wrong list of versions and just fix the expectation in the second commit.